ui/components/TroubleshootingComponent.js

```diff
@@ -1,24 +1,86 @@
 import * as React from 'react';
 import Button from '@material-ui/core/Button';
 import TroubleshootingModal from './TroubleshootingModalComponent';
+import Modal from './Modal';
+import { helpAndSupportModalSchema, helpAndSupportModalUiSchema } from '@layer5/sistent';
+import { useNotification } from '@/utils/hooks/useNotification';
+import { useGetLoggedInUserQuery } from '@/rtk-query/user';
+import SupportIcon from '@/assets/icons/support';
+import axios from 'axios';
+import { EVENT_TYPES } from 'lib/event-types';
 
 const Troubleshoot = (props) => {
   const [open, setOpen] = React.useState(true);
+  const [openForm, setOpenForm] = React.useState(false);
+  const { notify } = useNotification();
   const handleOpen = () => {
     setOpen(true);
   };
 
+  const { data: userData } = useGetLoggedInUserQuery();
+
+  const handleOpenForm = () => {
+    setOpenForm(true);
+  };
+
+  const handleSupportFormClose = () => {
+    setOpenForm(false);
+  };
+
+  const handleSupportFormSubmission = async (data) => {
+    try {
+      await axios.post('api/extensions/api/webhook/support', {
+        memberFormOne: {
+          ...data,
+          name: userData?.first_name + ' ' + userData?.last_name,
+          email: userData?.email,
```

Comment:
I think this is not correct payload right @sudhanshutech ? validation will be failing

Reply:
Please make sure we are passing same payload that we were doing in cloud to make sure validation get's passed

Reply:
its fixed and payload is now correct
![image](https://github.com/user-attachments/assets/ae712091-dc68-444a-8e86-94678c74eed5)
![image](https://github.com/user-attachments/assets/ca411633-4872-48a3-98a3-ce0e89562e29)